charts/kubescape-operator/values.yaml

```
@@ -112,6 +112,7 @@ global:
   networkPolicy:
     enabled: false
     createEgressRules: false
+  runtimePath: ""
```

```suggestion
  overrideRuntimePath: ""
```
and I would add some explanations to https://github.com/kubescape/helm-charts/blob/main/charts/kubescape-operator/README.md about how to find the correct runc path and where (here!) to set it